README.md

```diff
@@ -32,6 +32,21 @@ before forking.
 
 For the backstory on how Hamilton came about, see the original Stitch Fix [blog post!](https://multithreaded.stitchfix.com/blog/2021/10/14/functions-dags-hamilton/).
 
+# Feature comparison
+Here are common things that Hamilton is compared to, and how Hamilton compares to them.
+
+| Feature                                   | Hamilton | Macro orchestration systems like Airflow | Feast | DBT | Pandas |
```

Comment:
Are prefect, dagster, etc... all in `Macro orchestration systems like airflow`?

Reply:
yes.